src/Sparrow/Json/BlittableJsonReaderObject.cs

```diff
@@ -26,6 +26,10 @@ public unsafe class BlittableJsonReaderObject : BlittableJsonReaderBase, IDispos
         private readonly bool _isRoot;
         private byte* _objStart;
 
+        #if DEBUG || VALIDATE
```

Comment:
Is this debug leftovers?

Reply:
yes, but I'd like to leave this - as it may be useful for debugging memory issues in the future

Reply:
No, lte's remove that, debug builds are already expensive enough